XenOvfTransport/Export.cs

```diff
@@ -329,6 +329,11 @@ private EnvelopeType _export(Session xenSession, string targetPath, string ovfna
                 {
                     OVF.AddOtherSystemSettingData(ovfEnv, vsId, "hardware_platform_version", vm.hardware_platform_version.ToString(), OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_1"));
                 }
+
+                if (vm.HasSriovRecommendation())
+                {
+                    OVF.AddOtherSystemSettingData(ovfEnv, vsId, "allow_network_sriov", Messages.TRUE, OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_1"));

```

Comment:
How is this "allow_network_sriov" value used  in the import?

Reply:
They will be used in XAPI part. XAPI will know if the vm allow sriov.

Reply:
But I don't think that xapi ever reads the ovf file. The OVF import code is all in XenCenter.

Reply:
Agreed to do the import part in a separate ticket